src/backend/vulkan/src/lib.rs

```diff
@@ -318,20 +346,17 @@ impl core::Adapter<Backend> for Adapter {
         // Create associated command queues for each queue type
         let queues = queue_infos.iter().flat_map(|info| {
             (0..info.queue_count).map(|id| {
-                let queue = unsafe {
+                let queue_raw = unsafe {
                     device.raw.0.get_device_queue(info.queue_family_index, id)
                 };
-                unimplemented!()
-                /*
-                // TODO:
+                let queue = CommandQueue {
```

Comment:
I assume we still need to differ between queue types here?

Reply:
I'm on it: https://github.com/gfx-rs/gfx/pull/1449/commits/4fab52897c6d65107c745bf20532c2896142ed5e